user/user_password.php

```diff
@@ -38,7 +38,7 @@
 
 	if (!$msg->containsErrors()) {
 		// insert into the db.
-		$password   = $addslashes($_POST['form_password_hidden']);
+		$password   = addslashes($_POST['form_password_hidden']);
```

Comment:
Shall this `addslashes` be replaced?

Reply:
I wanted to ask you about that  part in the next meeting, other similar part are in profile/change_password.php (Line 69)
password_remainder.php (136)

I am confused as to what to used there, since passwords are only might to be hash, and they're not supposed to use addslashes, filter_var or htmlentities...

Reply:
I think I would go with ```$password = $_POST['form_password_hidden']```

Reply:
Sounds good to me. Thanks.

Reply:
OK, would do that

Reply:
Thanks, will check it out

Reply:
Please, what is the maximum length of the md5 function used is hashing the password, become online it is said that the maximum length for a  normal md5 function should be 32 characters, but it seems that of achecker used 40 characters.

Reply:
Yes, AChecker uses [sha-1](https://en.wikipedia.org/wiki/SHA-1) that produces 40 digit long hexadecimal.

Reply:
Ok, thanks